src/main/java/frc/robot/RobotContainer.java

```diff
@@ -204,31 +203,22 @@ public Command getAutonomousCommand() {
       new RamseteCommand(
           trajectory,
```

Comment:
Since trajectory could be null from the try-catch above, it would be good practice to check that it exists first. Add it to the driveBaseSubsystem check